python/tidy.py

```diff
@@ -39,6 +39,7 @@
     os.path.join(".", "target", "*"),
     os.path.join(".", "ports", "gonk", "src", "native_window_glue.cpp"),
     os.path.join(".", "ports", "cef", "*"),
+    os.path.join(".", "ports", "geckolib", "*"),
```

Comment:
Good catch - that was an accidental leftover from rebasing this over what actually landed in #9209.